src/registrar/templates/domain_request_form.html

```diff
@@ -16,6 +16,26 @@
             <use xlink:href="{%static 'img/sprite.svg'%}#arrow_back"></use>
           </svg><span class="margin-left-05">Previous step</span>
         </a>
+        {% comment %}
+        TODO: uncomment in #2596
```

Comment:
lol -- working ahead are we?  (I actually like doing this, too.  So hard to remember things after the fact sometimes)